docs/local_install_skaffold.md

```diff
@@ -186,3 +414,157 @@ org-2:
+- tiller issues: <https://stackoverflow.com/questions/51646957/helm-could-not-find-tiller#51662259>
+- After running `skaffold dev` in the `hlf-k8s` repo, in case of error related to the `tempchart` folder, please do `rm -rf tempchart`
+
+
+### [WIP] Ongoing issues
+
+- Unknown Host Request Forbidden on node 1...
+```sh
+kubectl edit deployment --namespace kube-system nginx-ingress-controller
+
+# in spec / container / args 
+# ADD 
+- --enable-ssl-passthrough
+# KO method
+```
+  - Possible alternative to nginx ingress: https://github.com/helm/charts/tree/master/stable/traefik
+  - Possible alternative to fix node-1 not being resolved from Clément:
+```sh
+minikube addons disable ingress
+helm install stable/nginx-ingress \
+  --name ingress \
+  --namespace kube-system  \
+  --set "controller.image.tag=0.27.1"  \
+  --set "controller.extraArgs.enable-ssl-passthrough="  \
+  --set-string "controller.config.http-redirect-code=301"  \
+  --set controller.hostNetwork=true  \
+  --set "controller.extraArgs.report-node-internal-ip-address=" \
+  --set "controller.image.runAsUser=101"
+  ```
+- If you are getting a `403` error only on <http://substra-backend.node-1.com/> with Firefox, please try with another browser...
+
+## 6. Further resources
+
+- Use [k9s](https://github.com/derailed/k9s):
+  - `CTRL + A`
+  - `:xray deployments all`
+  - `?` for help
+  - `/server` then `l` for the logs
+  - `y` to see the YAML configuration
+- kubectx & kubens: <https://github.com/ahmetb/kubectx#installation>
+- Local Kubernetes deployment with minikube: <https://kubernetes.io/blog/2019/03/28/running-kubernetes-locally-on-linux-with-minikube-now-with-kubernetes-1.14-support/>
+- Helm: <https://www.linux.com/tutorials/helm-kubernetes-package-manager/>
+- Hyperledger Fabric
+  - Installation: <https://medium.com/hackernoon/hyperledger-fabric-installation-guide-74065855eca9#c566>
+  - Building your first network: <https://hyperledger-fabric.readthedocs.io/en/release-1.4/build_network.html>
+- [Awesome Kubernetes list](https://github.com/ramitsurana/awesome-kubernetes#starting-point)
+- [Minikube](https://minikube.sigs.k8s.io/) is recommended but you can also use [Microk8s](https://microk8s.io/).
+- Use [Firefox Multi-Account Containers](https://addons.mozilla.org/en-US/firefox/addon/multi-account-containers/) extension to have several simultaneous different logins
+- TLDR (*Too Long; Didn't Read*): `sudo apt install tldr`
+
+## 7. Need help
+
+Let's talk:
+  - [Create an issue on Github](https://github.com/SubstraFoundation/substra/issues/new)
+  - Come to chat with us on [Slack](https://substra-workspace.slack.com/archives/CT54J1U2E)
+  - Have a look to the [forum](https://forum.substra.org/)
+  - Drop us an [email](mailto:help@substra.org)
+  - Or come meet us *afk* in Paris, Nantes or Limoges!
```

Comment:
```suggestion
  - Or come meet us *irl* in Paris, Nantes or Limoges!
```

Reply:
But, but, what happens behind a keyboard *is* happening *in real life*!

Reply:
Technically yes but the meaning of "afk" is mainly to express that you are "unavailable" ;)

Reply:
Okay!